modules/openshift-developer-cli-odo-getting-support.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-release-notes.adoc
+
+[id="odo-getting-support_{context}"]
+= Getting Support
+
+.Documentation:
```

Comment:
Shouldn't use a colon at the end when you're doing a title heading (.Documentation).

Same for the two headings below.